modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,298 @@
+// Module included in the following assemblies:
+//
+// * machine_management/
+// * machine_management/
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-node-custom-partition_{context}"]
+= Optional: Adding a new {op-system} worker node with a custom `/var` partition in AWS
+
+{product-title} supports partitioning devices during installation by using machine configs that are processed during the bootstrap. However, if you use `/var` partitioning, the device name must be determined at installation and cannot be changed. This means that you cannot add different instance types as nodes if they have different device naming schema. For example, if you configured the `/var` partition with the default AWS device name for `m4.large` instances, `dev/xvdb`, you cannot directly add an AWS `m5.large` instance, as it uses the `/dev/nvme1n1` device by using only a new machine set.   
+
+The procedure in this section shows how to add a new {op-system-first} compute node with an instance that uses a different device name from what was configured at installation. You will create a custom user data secret and configure a new machine set. These steps are specific to an AWS cluster.
```

Comment:
We can maybe consider adding at the end, "The principle applies to other cloud deployments as well, but the device naming schema would be different and should be determined on a per-case basis."

If we want to just document for AWS (as that was the original platform the user request came in on, what we have is fine too